index.html

```diff
@@ -1402,7 +1410,7 @@ <h2>Chunks</h2>
       <section id="4Concepts.FormatTypes">
         <h2>Chunk types</h2>
 
-        <p>There are 19 chunk types defined in this International Standard. Chunk types are four-byte sequences chosen so that they
+        <p>There are 24 chunk types defined in this International Standard. Chunk types are four-byte sequences chosen so that they
```

Comment:
I suggest removing this sentence altogether. Why is the number of chunk types important?

Reply:
Agreed. Will remove.